src/table.ts

```diff
@@ -764,15 +772,15 @@ class Table {
    * @param {function} callback The callback function.
    */
   private _mutate(
-      method: string, rows: object|object[], callback: s.CommitCallback): void {
-    this.database.runTransaction((err, transaction) => {
+      method: string, rows: object|object[], callback: CommitCallback): void {
+    this.database.runTransaction((err: Error|null, transaction?: Transaction|null) => {
```

Comment:
Why can't these types be inferred?

Reply:
Not exactly sure - VS Code is signaling implicit `any`